<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateTaskDialog</class>
 <widget class="QDialog" name="CreateTaskDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Создание задачи</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Название</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="nameEdit">
        <property name="whatsThis">
         <string>Название задачи</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Проверочные входные</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTextEdit" name="inputEdit">
        <property name="whatsThis">
         <string>Входные данные в программу для проверки на решение.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Проверочные выходные</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QTextEdit" name="outputEdit">
        <property name="whatsThis">
         <string>Данные с которыми сверяется ответ программы.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>HTML ресурс</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="pathEdit">
        <property name="whatsThis">
         <string>Путь ресурса</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openHTML">
        <property name="whatsThis">
         <string>Ресурс должен лежать в папке /db</string>
        </property>
        <property name="text">
         <string>Открыть</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="createButton">
     <property name="whatsThis">
      <string>Создать задачу в выбранном разделе</string>
     </property>
     <property name="text">
      <string>Создать задачу</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
